packages/autocomplete-js/src/__tests__/autocomplete.test.ts

```diff
@@ -156,6 +155,162 @@ describe('autocomplete-js', () => {
     `);
   });
 
+  test('renders empty template on empty results', async () => {
```

Comment:
We should limit our "empty" usage because we'll already have "empty template" and "renderEmpty".

```suggestion
  test('renders empty template on no results', async () => {
```